docker-compose/iceberg/docker-compose.yml

```diff
@@ -87,7 +87,7 @@ services:
   mc:
     depends_on:
       - minio
-    image: minio/mc
+    image: minio/mc:RELEASE.2022-12-24T15-21-38Z
```

Comment:
Hi @vuldin -- we recently did a similar update to our docker compose file in our Self-managed quickstart doc (see https://github.com/redpanda-data/docs/pull/1148/files#diff-eff275ccf349dd2f4bf2dd6ba9fc3612f7a81e3d06a922c925c47d2f33afe6a3R356 and https://docs.redpanda.com/current/get-started/quick-start/#deploy-redpanda-self-managed), I think it would make sense to keep things consistent, would you mind double checking that this works for you too?